src/availability/query_data.rs

```diff
@@ -190,8 +190,8 @@ pub trait QueryablePayload<Types: NodeType>: traits::BlockPayload<Types> {
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
 #[serde(bound = "")]
 pub struct LeafQueryData<Types: NodeType> {
-    pub(crate) leaf: Leaf<Types>,
-    pub(crate) qc: QuorumCertificate<Types>,
+    pub(crate) leaf: Leaf2<Types>,
```

Comment:
hmm I am not too sure about this change. We should be inserting Leaf1 for now or migrate the exisitng leaves to Leaf2 and start inserting leaf2

Reply:
but in the sequencer, we are inserting V1 of everything so I think we want to keep it as Leaf1?